components/BCRegistriesAccount.vue

```diff
@@ -39,7 +39,7 @@
           <div>
             <a
               class="link"
-              :href="$config.businessURL"
+              :href="appendAccountId($config.businessURL)"
```

Comment:
This appends `?accountid=xxx` to the URL, and is needed for the target URL to sync account ID.